### PR TITLE
Substations

### PR DESCRIPTION
**Pull request by andreagrandi**
* Add Substation model
* Add basic import script for Substation

### CODE REVIEW

**Comment by rossjones** [comment]
Is this ready to go? (Still marked WIP).

**Comment by andreagrandi** [comment]
@rossjones yes, still a WIP. I need to write the integration with Location model (I would prefer to do it in this PR).

File: landavailability/api/management/commands/import_substations.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        substation.operating = record.record[1]
+        substation.action_dtt = record.record[2]
+        substation.status = record.record[3]
+        substation.description = record.record[4]
+        substation.owner_flag = record.record[5]
+        substation.gdo_gid = record.record[6]
+        print(record.shape.__geo_interface__)
+        print(record.shape.shapeType)
+        substation.geom = GEOSGeometry(
+                json.dumps(record.shape.__geo_interface__), srid=3857)
```
Comment (andreagrandi) [inline landavailability/api/management/commands/import_substations.py]: p.s: before merging, let's double check I used the correct ```srid```

File: landavailability/api/management/commands/import_substations.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from .importers import ShapefileImportCommand
+import json
+
+
+class Command(ShapefileImportCommand):
+    help = 'Import Substations from a *.shp file'
+
+    def process_record(self, record):
+        if record.shape.__geo_interface__['type'] == 'Polygon':
+            print(record.record)
```
Comment (rossjones) [inline landavailability/api/management/commands/import_substations.py]: Necessary?

**Pull request merged**